packs/qualys/actions/lib/parsers.py

```diff
@@ -0,0 +1,39 @@
+from qualysapi.api_objects import Host, Scan, Report
+
+
+class FieldLists():
```

Comment:
Old style class. Didn't linter complain? http://stackoverflow.com/questions/36932/how-can-i-represent-an-enum-in-python